.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: php
-dist: precise
```

Comment:
I think we should add dist: trusty to not have the same surprise as we did when they changed the default to trusty.